api_client/python/timesketch_api_client/version.py

```diff
@@ -14,7 +14,7 @@
 """Version information for Timesketch API Client."""
 
 
-__version__ = '20201029'
+__version__ = '20201103'
```

Comment:
@kiddinn is that something you expect in a PR to bump up or do seperately?

Reply:
I would do this last, and yes I need this to be bumped before releasing a new version, but if you do this now you can expect some merge issues, since this file gets often updated.

So either do this last, or in a follow up pr

Reply:
ACK, removed it